ui-modules/app-inspector/app/views/main/inspect/summary/summary.template.html

```diff
@@ -86,11 +86,19 @@ <h2 ng-if="vm.entity" class="entity-name">
 
         <div ng-if="vm.location.items">
             <div ng-if="vm.location.items.length !== 0 || !vm.location.multi">
-                <p>This entity is running on <em><ng-pluralize count="vm.location.items.length" when="{'0': 'no locations', 'one': '{} location', 'other': '{} locations'}"></ng-pluralize></em></p>
+                <p>This entity is associated with <em><ng-pluralize count="vm.location.items.length" when="{'0': 'no locations', 'one': 'this location {} ', 'other': 'these locations'}"></ng-pluralize></em></p>
                 <ul>
                     <li ng-repeat="item in vm.location.items track by item.id">
-                        <strong>{{item.config['spec.final'] | specToLabel:vm.metadata}}</strong>
-                        <span ng-if="item.config.availabilityZone">in <code>{{item.config.availabilityZone}}</code> availability zone</span>
+                        <span ng-if="item.id"> Location Id: <strong>{{item.id | specToLabel:vm.metadata}}</strong> </span>
+                        <ul>
+                            <li ng-if="item.name"> <span> Name:  {{item.name | specToLabel:vm.metadata}}</span> </li>
+                            <li ng-if="item.config['spec.final']"> <span> Environment:  {{item.config['spec.final'] | specToLabel:vm.metadata}}</span> </li>
+                            <li ng-if="item.spec"> <span> Spec:  {{item.spec | specToLabel:vm.metadata}}</span> </li>
+                            <li ng-if="item.type"> <span> Type:  {{item.type | specToLabel:vm.metadata}}</span> </li>
+                            <li ng-if="item.config['address']"> <span> Address:  {{item.config['address'] | specToLabel:vm.metadata}}</span> </li>
+                            <li ng-if="item.config['endpoint']"> <span> Endpoint:  {{item.config['endpoint'] | specToLabel:vm.metadata}}</span> </li>
```

Comment:
Should be equivalent to `item.config.address` and `.endpoint`.

Reply:
@jathanasiou I actually like the `item.config['address']` syntax better. Is there any advantage to using one or the other ?

Reply:
@iuliana not really but the dot syntax is considered the standard for JSON-like structures. They have no functional difference but essentially
- dots look simpler
- brackets allow for complex or dynamic keys eg `myObj[getKey()]`

Brackets basically hint that you're either accessing an array or have complex index keys and need that syntax. For the sake of uniformity I consider dots better with brackets used only on these exceptional cases, but that's a bit subjective .

Reply:
@jathanasiou  I also noticed that `[]` syntax is needed when property names have dots in them. So the a`item.config['spec.final']` stays. :)